src/bootstrap/server-config.js

```diff
@@ -17,15 +17,21 @@ const configSchema = joi.object({
   sessionSecret: joi.string(),
   sessionDurationInMinutes: joi.number().min(1),
   smtpOptions: joi.any(),
-  emailSenderAddress: joi.string(),
+  emailSenderAddress: joi.string().require(),
```

Comment:
I made this required as I think it kind of is, isn't it?
Shall we keep it and I fix the texts or revert?

Reply:
It's good to have it, I think

Reply:
That might explain some things 😂 Thanks